### ISSUE TITLE
Support Django 3.0

### ISSUE THREAD

**Issue by snake-soft** [django-oscar/django-oscar-paypal#255]
In Django 3 they dropped support for Python2 compatibility APIs:
[https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis](https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis)

Because of that django-oscar-paypal threw an error when using it with Django3:
`ImportError: cannot import name 'python_2_unicode_compatible' from 'django.utils.encoding' (/home/me/.virtualenvs/oscar/lib/python3.7/site-packages/django/utils/encoding.py)`

I found the solution for that problem here:
[https://forum.djangoproject.com/t/python-2-unicode-compatible/715](https://forum.djangoproject.com/t/python-2-unicode-compatible/715)

In short - The compatibility api is available as standalone package:
[https://pypi.org/project/six/](https://pypi.org/project/six/)

---

Are there any other issues known that could avoid the django 3 compatibility?

**Comment by Dihfahsih1** [comment]
To install oscar with django 3.x, do the following:

```
pip3 install django-tables2
pip3 install django-oscar==3.0a0
pip3 install Django==3.1.3
```

**Comment by Jihakuz** [comment]
The latest version of django-oscar-paypal seems to fix the issue, but it is not in pypi yet.

`pip3 install git+https://github.com/django-oscar/django-oscar-paypal.git` fixed the issue for me.
